src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -269,12 +272,14 @@ CameraTrigger::CameraTrigger() :
 	_p_activation_time = param_find("TRIG_ACT_TIME");
 	_p_mode = param_find("TRIG_MODE");
 	_p_interface = param_find("TRIG_INTERFACE");
+	_p_cam_cap_fback = param_find("CAM_CAP_FBACK");
```

Comment:
I need to better understand the overall design, but reaching into other modules params is something I'd like to avoid.

Reply:
@dagar I agree. The reason for this is because when the camera_capture driver is active I rename the topic name of the camera_trigger  driver so that the camera_capture can publish the camera_trigger message (which is the one that goes to the geotagging and to mavlink). Do you have any idea how this could be cleaned up? In any case, even if we want to keep this structure I think the parameter should be defined in the camera_trigger driver because otherwise this will cause issues on the targets that won't have the camera_capture driver in their cmake.